### DISCUSSION TITLE
Cant create and save a model in the radio TX16s MKII

### DISCUSSION THREAD

**Discussion by jacksoderberg** [EdgeTX/edgetx#2708] [Q&A]
Hello
As title, upon creating a model in the radio as soon the radio is turned of the binding is lost, when hooking up the radio to companion models can be created and saved as normal, what Im I missing?

Cheers
Jack

**Comment by pfeerick** [comment]
Did you get anywhere with this? What firmware version was this with? Have you tried updating to 2.8.0?

**Reply by jacksoderberg** [reply]
unfortunately not, mine firmware version is 2.7.1 Black pearl, I have not tried to upgrade since im kind a afraid to mess upp the radio, mine radio is working fine beside this issue, I have created som empty models in companion.

Cheers
Jack
